### DISCUSSION TITLE
my links dont work

### DISCUSSION THREAD

**Discussion by beeswithchainsaws** [cifkao/html-midi-player#67] [Q&A]
ok so I've been trying to get this to work for my website, and it seems pretty straightforward, but every time i add my own link to it, it just doesn't work anymore. I haven't done any HTML stuff before, so if someone could tell me what the attached image means, I'd really appreciate it.
![sdvdfsgvwv](https://github.com/cifkao/html-midi-player/assets/137239245/6a0896bb-22d9-49f3-b456-34a4d9d939d0)

**Comment by cifkao** [comment]
Please check [here](https://github.com/cifkao/html-midi-player/tree/master#why-is-my-midi-file-not-loading) the corresponding section of the FAQ that I just added. Hope it solves your problem.

**Reply by beeswithchainsaws** [reply]
ok i checked the error, and it told me "failed to fetch"

**Reply by cifkao** [reply]
If that doesn't help, then you should check the logs of your HTTP server or the response in Developer Tools to see what the problem is. I am not able to help if I don't know your setup.
